homeassistant/components/starline/services.yaml

```diff
@@ -0,0 +1,10 @@
+update_state:
+  description: >
+    Fetch the last state of the devices from the StarLine server.
+set_scan_interval:
```

Comment:
We should not offer this service, please remove.

Reply:
The same reason. Users want to update data more often during the day, and less often at night. I agree with them - there is API limit 1000 requests per day and it's not so important to update data often at night, but it's good to update more often during the day.